exporter/clients_test.go

```diff
@@ -1,16 +1,24 @@
 package exporter
 
 import (
+	"strconv"
 	"strings"
 	"testing"
 
 	"github.com/prometheus/client_golang/prometheus"
 )
 
 func TestParseClientListString(t *testing.T) {
+	createdAt6321, _ := durationFieldToTimestamp("6321")
```

Comment:
Instead of having these called one-off here can you add a new `Test...` function that loops through a bunch of test cases and directly calls and tests `durationFieldToTimestamp()` ?

Reply:
(That would/should also address the comments left by the bot about the lines not covered by tests)

Reply:
proper testing of `durationFieldToTimestamp` would require frozen time. I think this function is small enough to ignore it being not tested.
I improved `TestParseClientListString` as much as possible.

Reply:
What I mean is using table driven tests which are common in Go.
Add tests for all the corner cases to improve test coverage.

See e.g. this one here in the std lib: https://golang.org/src/time/time_test.go#L1003

and this longer explanation: https://mj-go.in/golang/table-driven-tests-in-go